share/tools/web_config/sample_prompts/classic_status.fish

```diff
@@ -3,10 +3,23 @@
 
 function fish_prompt --description "Write out the prompt"
     # Save our status
+    set -l last_pipestatus $pipestatus
     set -l last_status $status
 
+    # only output $pipestatus if there was a pipe and any part of it had non-zero exit status
+    # TODO maybe have a common function that returns true if all array elements match a certain value?
```

Comment:
I mean... `string match` can be used - `string match -rvq '^0$' -- $last_pipestatus`. It's an inverted ("-v") match for exactly 0, and it's quiet. So it'll return true if something wasn't 0.

But in general doing a boring `for`-loop isn't _bad_. It's nice and easy and understandable.

Reply:
I really like that, thank you. It simplified it a lot reducing 2 levels of indentation!